src/main/java/uk/ac/ebi/atlas/experimentpage/differential/topgenes/DifferentialExperimentTopGenesDao.java

```diff
@@ -32,8 +32,8 @@ public TupleStreamer createForDifferentialNonSpecific(String experimentAccession
 //                        experimentAccession, preferences);
```

Comment:
Are those `solrQuery` still useful or we could delete them? This results in a stale field `experimentAccession`.